pcdsdevices/epics_motor.py

```diff
@@ -1523,6 +1523,58 @@ def __init__(self, prefix='', *, tip_pv, tilt_pv, **kwargs):
         super().__init__(prefix, **kwargs)
 
 
+class SmarActPicoscale(SmarAct):
+    """
+    Class for encoded SmarAct motors controller via the MCS2 controller
+    which use the PicoScale sensor heads for encoded positions.
+    Instantiating a device requires inclusion of the ioc_base in order to
```

Comment:
Just making sure: ioc_base is something that would be obvious to someone who deploys smaract IOCs? I can think of two different things this can mean in abstract (the IOC admin prefix, a controller-based prefix)

Reply:
The latter not the former. IIRC the former has `IOC:<ioc_base>` format using my current meaning of `ioc_base`

Reply:
I'm trying to make sure someone will be able to follow the docs later, so my suggestion is to consider if it needs clarification, or if it does not. Otherwise the PR looks good to me!

Reply:
Definitely valid to ask! I'm actually _not_ entirely sure we're using this field as intended, but it certainly _is_ how we're using it in existing devices in MODS:
![image](https://github.com/pcdshub/pcdsdevices/assets/149725219/f3f6f7ee-00a9-4849-bf10-aa4a1a20b129)

I tried to pick what already existing in the `happi` container